string name
bool activity_status